src/util/timeConverterToMinSec.ts

```diff
@@ -0,0 +1,14 @@
+export function timeConverterToMinSec(milliseconds: number) : string {
+
+    let minutes: number = Math.floor(milliseconds/ (1000 * 60) );
+    let seconds: number = Math.floor((milliseconds / 1000) % 60);
+    let formattedSeconds: string;
+
+    if (seconds < 10) {
+        formattedSeconds = "0" + seconds;

```

Comment:
You probably need a clause like this for mins as well. Add a test first to cause this to fail and then fix it up here.

Reply:
Careful with types. Seconds here is a number which you’re adding onto a string.